src/common/Button.jsx

```diff
@@ -26,10 +24,14 @@ const Buttons = props => (
     *@property {String} className - The ClassName of the button for syling
     *@property {String} value - The name to show in the button
   */
-Buttons.propTypes = {
+Button.propTypes = {
   name: PropTypes.string.isRequired,
   className: PropTypes.string.isRequired,
   value: PropTypes.string.isRequired,
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
```

Comment:
rename this to `handleClick` to be consistent with its usage in other components